cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -930,8 +930,7 @@ runTransactionSignCmd
       inputTxFile <- liftIO $ fileOrPipe inputTxFilePath
       anyTx <- lift (readFileTx inputTxFile) & onLeft (left . TxCmdCddlError)
 
-      InAnyShelleyBasedEra sbe tx <-
-          onlyInShelleyBasedEras "sign for Byron era transactions" anyTx
+      InAnyShelleyBasedEra sbe tx <- pure anyTx
```

Comment:
We were never supported Byron era tx signing. Indication that people were using the "old" `cardano-cli byron` cli commands for everything Byron related.